SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -266,6 +266,16 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 
 	static void ResetShadowData(FRepLayout& RepLayout, FRepStateStaticBuffer& StaticBuffer, UObject* TargetObject);
 
+	void SetNeedOwnerInterestUpdate(bool bInNeedOwnerInterestUpdate)
+	{
+		bNeedOwnerInterestUpdate = bInNeedOwnerInterestUpdate;
+	}
+
+	bool NeedOwnerInterestUpdate() const
```

Comment:
Does need mean awaiting? If so we could change the name to that as I got confused about this at first.

Reply:
Yes, we are awaiting for our owner to get an entity id, but what it really means is that our interest is out of date.